pkg/pillar/hypervisor/hypervisor.go

```diff
@@ -5,14 +5,14 @@ package hypervisor
 
 import (
 	"fmt"
-	"os"
-	"strings"
-
+	"github.com/lf-edge/eve/pkg/pillar/base"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	uuid "github.com/satori/go.uuid"
 	"github.com/shirou/gopsutil/cpu"
 	"github.com/shirou/gopsutil/mem"
 	"github.com/sirupsen/logrus"
+	"os"
+	"strings"
```

Comment:
These were moved to the wrong place. Should have standard libraries up top.

Reply:
Will fix that.

Reply:
+1

Reply:
I fixed those in my next commit but waiting for PR https://github.com/lf-edge/eve-api/pull/55 to be approved/merged before I push next commit. Else, compilation will fail.